### PR TITLE
Update gnomad variant pulling script for v4

### PR DESCRIPTION
**Pull request by rileyhgrant**
Add the option to pull v4 variants in the script.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Variant Curation Portal
 
-![CI status bage](https://github.com/macarthur-lab/variant-curation-portal/workflows/CI/badge.svg)
+![CI status bage](https://github.com/broadinstitute/variant-curation-portal/actions/workflows/frontend-ci.yml/badge.svg)
```
Comment (phildarnowsky-broad) [inline README.md:3]: Should "bage" be "badge"?

File: scripts/get_gnomad_lof_variants.py [hunk 2]
```diff
@@ -134,30 +185,41 @@ def add(a, b):
...
+                f"{chrom_prefix}{gene['chrom']}:{gene['start']}-{gene['stop']}",
+                reference_genome=reference_genome,
             )
             for gene in genes
         ],
     )
 
     gene_ids = hl.set(gene_ids)
 
+    if gnomad_version == 2:
```
Comment (phildarnowsky-broad) [inline scripts/get_gnomad_lof_variants.py:218]: Let's cut this branch and the `annotate_gnomad_v2_etc` function it calls until such time, which is as you note above possibly never, that we actually need it.

File: scripts/get_gnomad_lof_variants.py [hunk 3]
```diff
@@ -178,7 +240,13 @@ def get_gnomad_lof_variants(gnomad_version, gene_ids, include_low_confidence=Fal
     ds = ds.select(
         reference_genome="GRCh37" if gnomad_version == 2 else "GRCh38",
         variant_id=variant_id(ds.locus, ds.alleles),
-        liftover_variant_id=hl.missing(hl.tstr),
+        liftover_variant_id=hl.if_else(
```
Comment (phildarnowsky-broad) [inline scripts/get_gnomad_lof_variants.py:243]: It's hard to follow these multiple `is_missing` and `if_else` calls here, introducing a few intermediate variables might be the thing to clarify this bit.

File: curation_portal/views/variants.py [hunk 4]
```diff
@@ -12,14 +12,13 @@ class VariantsView(APIView):
...
-                Q(reference_genome="GRCh37")  # TODO: Handle different reference genomes
-                & (
-                    Q(project__owners__id__contains=request.user.id)
-                    | Q(curation_assignment__curator=request.user)
-                )
+                Q(project__owners__id__contains=request.user.id)
+                | Q(curation_assignment__curator=request.user)
             )
-            .values_list("variant_id", flat=True)
+            .values_list("variant_id", "reference_genome")
```
Comment (phildarnowsky-broad) [inline curation_portal/views/variants.py:18]: `flat=True` came out on purpose? IDK what that flag actually does, just making sure.

File: assets/components/Fetch.js [hunk 5]
```diff
@@ -10,11 +10,14 @@ class BaseFetch extends Component {
   static propTypes = {
     children: PropTypes.func.isRequired,
     path: PropTypes.string.isRequired,
+    // eslint-disable-next-line react/forbid-prop-types
```
Comment (phildarnowsky-broad) [inline assets/components/Fetch.js:13]: If you're going to go on with PropTypes for this codebase in the forseeable future, may as well just turn off this rule in the eslint config.

**Review by phildarnowsky-broad — CHANGES_REQUESTED** [review]
No major changes requested, just some loose ends.

File: README.md (hunk 1 above)
Comment (rileyhgrant) [reply README.md:3]: Oop yes haha, good eye.

File: assets/components/Fetch.js (hunk 5 above)
Comment (phildarnowsky-broad) [reply assets/components/Fetch.js:13]: Yeah my position is I would, all else being equal, prefer TS to Proptypes, but in this particular case there's not much to be gained from a TS migration and proptypes are better than no types at all.

File: scripts/get_gnomad_lof_variants.py [hunk 6]
```diff
@@ -181,6 +176,17 @@ def variant_id(locus, alleles):
     )
 
 
+def liftover_variant_id(exome_liftover_id, genome_liftover_id, alleles):
+    return variant_id(
+        hl.if_else(
+            hl.is_missing(exome_liftover_id),
+            genome_liftover_id,
```
Comment (phildarnowsky-broad) [inline scripts/get_gnomad_lof_variants.py:183]: Are both of these meant to be `genome_liftover_id` or is line 184 perhaps supposed to be `exome_liftover_id`?

**Review by phildarnowsky-broad — CHANGES_REQUESTED** [review]
One small likely bug flagged

File: scripts/get_gnomad_lof_variants.py (hunk 6 above)
Comment (rileyhgrant) [reply scripts/get_gnomad_lof_variants.py:183]: Ohp yes great catch, thank you! My mistake.
